app/components/NotificationItem.vue

```diff
@@ -43,22 +43,31 @@
                     columns="auto,auto"
                     @tap="toggleMenu"
                 >
-                    <Label col="0" :text="_L('dismiss')" class="action-btn" />
-                    <Image col="1" src="~/images/Icon_Menu_Down.png" width="8" class="m-l-2" :class="isAndroid ? 'm-t-2' : 'm-t-8'" />
+                    <Label col="0" :text="_L('dismiss')" class="action-btn"/>
+                    <Image col="1" src="~/images/Icon_Menu_Down.png" width="8" class="m-l-2"
+                           :class="isAndroid ? 'm-t-2' : 'm-t-8'"/>
                 </GridLayout>
+                <Label
+                    col="2"
+                    :text="prettyCreatedAt(notification.created)"
+                    horizontaAlign="right"
+                    class="notify-date"
+                />
             </GridLayout>
         </GridLayout>
-        <GridLayout rows="*,*" class="size-12 menu" horizontalAlignment="right" v-if="showMenu.includes(notification.id)">
-            <Label row="0" :text="_L('notificationRemindLater')" textWrap="true" class="bold p-10" @tap="dismiss" />
-            <Label row="1" :text="_L('notificationDontRemind')" textWrap="true" class="bold p-10" @tap="satisfy" />
+        <GridLayout rows="*,*" class="size-12 menu" horizontalAlignment="right"
+                    v-if="showMenu.includes(notification.id)">
+            <Label row="0" :text="_L('notificationRemindLater')" textWrap="true" class="bold p-10" @tap="dismiss"/>
+            <Label row="1" :text="_L('notificationDontRemind')" textWrap="true" class="bold p-10" @tap="satisfy"/>
         </GridLayout>
     </GridLayout>
 </template>
 <script lang="ts">
 import Vue from "vue";
-import { isAndroid, Label } from "@nativescript/core";
-import { _L } from "@/lib";
-import { fullRoutes } from "~/routes";
+import {isAndroid, Label} from "@nativescript/core";
+import {_L} from "@/lib";
+import {fullRoutes} from "~/routes";
+import moment from "moment";
```

Comment:
Can we some spaces inside the {}'s, please?

Reply:
Oops, looks like I didn't bind prettier correctly to my usual formatting shortcut.

This file should now be pretty.